SPECS/python-fields/python-fields.spec

```diff
@@ -0,0 +1,123 @@
+# Compatibility with python-sphinx >= 1.3, already applied upstream
+Patch0:         %{name}-5.0.0-sphinx-1.3.patch
+BuildArch:      noarch
+
+%description
+Container class boilerplate killer.
+
+Features:
+- Human-readable __repr__
+- Complete set of comparison methods
+- Keyword and positional argument support. Works like a normal class - you can
+  override just about anything in the subclass (eg: a custom __init__). In
+  contrast, hynek/characteristic forces different call schematics and calls
+  your __init__ with different arguments.
+
+%package doc
+Summary:        Documentation for '%{name}'
+BuildRequires:  python%{python3_pkgversion}-sphinx
+BuildRequires:  python%{python3_pkgversion}-sphinx-theme-py3doc-enhanced
+BuildRequires:  python%{python3_pkgversion}-sphinxcontrib-websupport
+
+%description doc
+HTML API documentation for the '%{srcname}' Python module.
+
+%package -n     python%{python3_pkgversion}-%{srcname}
+Summary:        %{summary}
+%{?python_provide:%python_provide python%{python3_pkgversion}-%{srcname}}
+BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
+BuildRequires:  python%{python3_pkgversion}-sphinx
+BuildRequires:  python%{python3_pkgversion}-sphinxcontrib-websupport
+BuildRequires:  python%{python3_pkgversion}-xml
+%if %{with_check}
+BuildRequires:  python3-pip
+%endif
+
+%description -n python%{python3_pkgversion}-%{srcname}
+Container class boilerplate killer.
+
+Features:
+- Human-readable __repr__
+- Complete set of comparison methods
+- Keyword and positional argument support. Works like a normal class - you can
+  override just about anything in the subclass (eg: a custom __init__). In
+  contrast, hynek/characteristic forces different call schematics and calls
+  your __init__ with different arguments.
+
+%prep
+%autosetup -p1
+sed -i 's/\[pytest\]/\[tool:pytest\]/' setup.cfg
+
+%build
+%py3_build
+PYTHONPATH=$PWD/src sphinx-build%{python3_pkgversion} -b html docs docs/_build/html
+rm -rf docs/_build/html/.buildinfo docs/_build/html/.doctrees
+
+%install
+%py3_install
+
+%check
+# Perf tests require unmaintained 'characteristic' module
+pip3 install atomicwrites>=1.3.0 \
+    attrs>=19.1.0 \
+    more-itertools>=7.0.0 \
+    pluggy>=0.11.0 \
+    pytest>=5.4.0 \
+    pytest-cov>=2.7.1 \
+    pytest-benchmark
+PATH=%{buildroot}%{_bindir}:${PATH} \
+PYTHONPATH=%{buildroot}%{python3_sitelib} \
+    python%{python3_version} -m pytest -v --ignore=tests/test_perf.py tests
+
+%files doc
+%license LICENSE
+%doc docs/_build/html
+
+%files -n python%{python3_pkgversion}-%{srcname}
+%license LICENSE
+%doc AUTHORS.rst
+%doc CHANGELOG.rst
+%doc README.rst
+%{python3_sitelib}/%{srcname}/
+%{python3_sitelib}/%{srcname}-%{version}-py%{python3_version}.egg-info/
+
+%changelog
+* Wed Jun 23 2021 Rachel Menge <rachelmenge@microsoft.com> - 5.0.0-9
+- Update cgmanifest and license info
```

Comment:
>Update cgmanifest [](http://example.com/codeflow?start=2&length=17)

Similar comment about the cgmanifest: now sure what this means in the context of this spec.